fiatconnect-api.md

```diff
@@ -1796,6 +1796,897 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountIfsc`
+
+`CheckingAccountIfsc` is a Fiat Account schema that represents accounts for transactions across `INR` currency.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "ifsc": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIfsc"
+}
+```
+####  7.3.2.3. `CheckingAccountIban`
+
+`CheckingAccountIban` is a Fiat Account schema that represents accounts for transactions across `EUR` currency.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "iban": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIban"
+}
+```
+
+####  7.3.2.4. `CheckingAccountBankCode`
+
+`CheckingAccountBankCode` is a Fiat Account schema that represents accounts for transactions across `NG`, `GH`, `KE`, `ZA`, `PH`, `SN`, `CN`, `GB`, `SG`, `TZ`, `HK` currencies.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "bankCode": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountBankCode"
+}
+```
+
+####  7.3.2.5. `CheckingAccountMobile`
+
+`CheckingAccountMobile` is a Fiat Account schema that represents accounts for transactions across `NG`, `GH`, `KE`, `UG`, `SN`, `BJ`, `ML`, `CI`, `NE`, `TG`, `GW`, `BF`, `SL`, `RW`, `CM`,`CG` currencies.
+
+```
+{
+    "accountName": `string`,
+    "mobileProvider": `string`,
+    "mobileProviderCode": `string`,
+    "mobileNumber": `string`,
+    "fiatAccountType": `CheckingAccountMobile`,
+    "country":'string'
+}
+```
+
+
+The `country` field should be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. The field is for providers to specify which country this data is meant for.
+
+####  7.3.2.2.1 `SupportedBankMobileProvidersEnum`
+
+For `NG` the list of supported mobile provider codes is as mentioned below -
```

Comment:
This list of all provider codes that are accepted per country shouldn't be in the FiatConnect spec IMO; it looks highly specific to BitSSA. Instead, it should be given to the client in a list of allowedValues in a response to the initial quote request, for which `country` is a parameter. (So BitSSA has the chance to tell the client which providers are accepted dynamically when they request a quote.)

This is actually good for BitSSA, because you can add or remove providers dynamically without having to first undergo a spec change (provided that the name of the provider is already listed in an enum in the specification somewhere)

Reply:
We had suggested we provide the list of bank names and bank codes in a getBankCode API call,  On the partner call we were told to include this in the schema.

Reply:
We will add the bank and mobile provider code to the response for quote out as allowed vaues.